adapters/appnexus/appnexusplatformtest/video/simple-video.json

```diff
@@ -1,132 +0,0 @@
-{
```

Comment:
Why did we need to remove this test?

Reply:
Scott, this unit test now fails due to random value in response. I haven't found a way to pass reg exp or override marshaler. I put a separate comment about it earlier. https://github.com/prebid/prebid-server/pull/1444#issuecomment-675747960
Now working on your other comments.

Reply:
Gotcha. Do the new unit tests cover the same functionality?

Reply:
I was thinking about this too. This test calls video endpoint with Appnexus bidder. This is equal to some tests in `video_auction_test.go`, where we call the entire video endpoint like this: `req := httptest.NewRequest("POST", "/openrtb2/video", strings.NewReader(reqBody))` and put Appnexus params in configIds:

```
var testVideoStoredImpData = map[string]json.RawMessage{
	"fba10607-0c12-43d1-ad07-b8a513bc75d6": json.RawMessage(`{"ext": {"appnexus": {"placementId": 14997137}}}`),
	"8b452b41-2681-4a20-9086-6f16ffad7773": json.RawMessage(`{"ext": {"appnexus": {"placementId": 15016213}}}`),
	"87d82a45-35c3-46cc-9315-2e3eeb91d0f2": json.RawMessage(`{"ext": {"appnexus": {"placementId": 15062775}}}`),
}
```

Do you think I need to add more tests? If yes, should this be a part of `video_action_test` or `appnexus_test`?